src/Lykke.Service.HFT/Services/StartupManager.cs

```diff
@@ -47,7 +47,9 @@ public async Task StartAsync()
                 realm.SessionClosed += (sender, args) => { _sessionRepository.TryRemoveSessionId(args.SessionId); };
             }
 
+#if !DEBUG
             await _apiKeyCacheInitializer.InitApiKeyCache();
```

Comment:
and this

Reply:
This is a long-term process that can influence other instances. I wouldn't like this code to run when I am debugging.